scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RuntimeReducerEstimator.scala

```diff
@@ -8,11 +8,40 @@ import org.apache.hadoop.mapred.JobConf
 import org.slf4j.LoggerFactory
 
 object RuntimeReducerEstimator {
-  val RuntimePerReducer = "scalding.reducer.estimator.runtime.per.reducer"
-  val defaultRuntimePerReducer = 10 * 60 * 1000 // 10 mins
 
```

Comment:
Any reason for removing these? Might be useful for outside code to fetch the defaults.